watertap/unit_models/translator_adm1_asm1.py

```diff
@@ -0,0 +1,463 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Modified translator block.
+
+This is copied from the Generic template for a translator block.
```

Comment:
Why not inherit from the translator block and build on top of it? I would imagine it would eliminate some unnecessary (duplicated) lines of code, though it wouldn't be necessary for functionality purposes I suppose.